### ISSUE TITLE
[new dezoomer] encrypted krpano

### ISSUE THREAD

**Issue by rmartin-rp** [lovasoa/dezoomify#293]
Hi @avindra 

From Krpano:

Uncaught TypeError: Cannot read property 'documentElement' of null
(http://ophir.alwaysdata.net/dezoomify/zoommanager.js:360)

URL:
https://tourmkr.com/t3OTe4LUqP?index=scene_101&hlookat=32&vlookat=14&fov=110

I think this also happen in other sites. 

Thanks for your  great work!

**Comment by lovasoa** [comment]
I cannot reproduce that. Which version of which browser are you using ?

**Comment by rmartin-rp** [comment]
I'm getting this error on Chrome 73.0.3683.103

Tried with Firefox 66.0.3 and the error is different:
TypeError: response is null 
(https://ophir.alwaysdata.net/dezoomify/zoommanager.js:382)

Both browsers are running under ubuntu 16.04 (also tried with kubuntu 17).

Anything else you need just let me know :)
Thanks

**Comment by lovasoa** [comment]
There was a bug in dezoomify that was hiding the underlying error (a failed network request).

**Comment by rmartin-rp** [comment]
I still getting the same error in your website. Just in case it wasn't updated I've just cloned the repository and I get this error:
Script error.
(:0)

Probably something I did bad. I've also found another URL where I get the same problem (in your server and in local):

https://www.everpano.com/

**Comment by lovasoa** [comment]
I did one more update. The error message should now be correct. You should now see :

 > Error: Invalid XML: https://www.everpano.com/

**Comment by rmartin-rp** [comment]
Thanks, I tried and I get now the message you said. I guess "Invalid XML" means I cannot get the image from that site.

**Comment by lovasoa** [comment]
No, they use an encrypted version of krpano that is not supported by dezoomify. If you want to work on supporting it, you are welcome.
You may also be interested by [krpano.tech](http://www.krpano.tech/en/): a chinese developer seems to already have worked on downloading krpano images, including ones with encryption enabled.

**Comment by lovasoa** [comment]
About the initial image (https://tourmkr.com/t3OTe4LUqP?index=scene_101&hlookat=32&vlookat=14&fov=110) : 

It looks like the tiles are generated dynamically by the server, and you can have the server return you the full original image:

https://img.gothru.org/cube/2912/17124382152032285257/20200226151849.xjh5ne_1.jpg?crop=0x0x6144x4096&resize=9999x&save=optimize,progressive,qual:90

![](https://img.gothru.org/cube/2912/17124382152032285257/20200226151849.xjh5ne_1.jpg?crop=0x0x6144x4096&resize=6144x&save=optimize,progressive,qual:70)

I also added support for krpano in [dezoomify-rs](https://lovasoa.github.io/dezoomify-rs/)

**Comment by AlexanderDeLargeBrotherSir** [comment]
@rmartin-rp, look into this again and if your issue is resolved then kindly close this.

**Comment by rmartin-rp** [comment]
The url not longer exists so I can't say it's resolved. Anyways, closing this one. Thanks for keeping an eye on it.
